bindings/python/pinocchio/robot_wrapper.py

```diff
@@ -262,12 +274,25 @@ def getViewerNodeName(self, geometry_object, geometry_type):
         """For each geometry object, returns the corresponding name of the node in the display."""
         return self.viz.getViewerNodeName(geometry_object, geometry_type)
 
-    def initViewer(self, *args, **kwargs):
+    def initViewer(self, shared_data=True, *args, **kwargs):
```

Comment:
I find that *share_data* is a better name.

Reply:
Done, could you check it again ?